src/internal/cache-client.ts

```diff
@@ -1640,6 +1643,172 @@ export class CacheClient {
     });
   }
 
+  public async sortedSetPutValue(
+    cacheName: string,
+    sortedSetName: string,
+    value: string | Uint8Array,
+    score: number,
+    ttl: CollectionTtl = CollectionTtl.fromCacheTtl()
+  ): Promise<CacheSortedSetPutValue.Response> {
+    try {
+      validateCacheName(cacheName);
+      validateSortedSetName(sortedSetName);
+    } catch (err) {
+      return new CacheSortedSetPutValue.Error(normalizeSdkError(err as Error));
+    }
+    this.logger.trace(
+      `Issuing 'sortedSetPutValue' request; value: ${value.toString()}, score : ${score}, ttl: ${
+        ttl.ttlSeconds.toString() ?? 'null'
+      }`
+    );
+
+    const result = await this.sendSortedSetPutValue(
+      cacheName,
+      this.convert(sortedSetName),
+      this.convert(value),
+      score,
+      ttl.ttlMilliseconds() || this.defaultTtlSeconds * 1000,
+      ttl.refreshTtl()
+    );
+    this.logger.trace(
+      `'sortedSetPutValue' request result: ${result.toString()}`
+    );
+    return result;
+  }
+
+  private async sendSortedSetPutValue(
+    cacheName: string,
+    sortedSetName: Uint8Array,
+    value: Uint8Array,
+    score: number,
+    ttlMilliseconds: number,
+    refreshTtl: boolean
+  ): Promise<CacheSortedSetPutValue.Response> {
+    const request = new grpcCache._SortedSetPutRequest({
+      set_name: sortedSetName,
+      elements: [new grpcCache._SortedSetElement({value, score})],
+      ttl_milliseconds: ttlMilliseconds,
+      refresh_ttl: refreshTtl,
+    });
+    const metadata = this.createMetadata(cacheName);
+    return await new Promise(resolve => {
+      this.clientWrapper.getClient().SortedSetPut(
+        request,
+        metadata,
+        {
+          interceptors: this.interceptors,
+        },
+        (err, resp) => {
+          if (resp) {
+            resolve(new CacheSortedSetPutValue.Success());
+          } else {
+            resolve(
+              new CacheSortedSetPutValue.Error(cacheServiceErrorMapper(err))
+            );
+          }
+        }
+      );
+    });
+  }
+
+  public async sortedSetFetchByIndex(
+    cacheName: string,
+    sortedSetName: string,
+    startIndex?: number,
+    endIndex?: number
+  ): Promise<CacheSortedSetFetch.Response> {
+    try {
+      validateCacheName(cacheName);
+      validateSortedSetName(sortedSetName);
+    } catch (err) {
+      return new CacheSortedSetFetch.Error(normalizeSdkError(err as Error));
+    }
+    //this.logger.trace(
+    //`Issuing 'sortedSetFetchByIndex' request; startIndex: ${startIndex}, endIndex : ${endIndex}`
```

Comment:
can use the logger varargs signature here to avoid the linting error

Reply:
Fixed in [49daeee](https://github.com/momentohq/client-sdk-nodejs/pull/314/commits/49daeeef75b3f569d3db576b7bf01e5ff5ae2923)